crates/fs/common/utils.rs

```diff
@@ -43,6 +46,23 @@ impl crate::private::Rng for TestRng {
     }
 }
 
+#[cfg(test)]
+impl ProptestRng {
+    pub(crate) fn from_seed(algorithm: proptest::test_runner::RngAlgorithm, seed: &[u8]) -> Self {
+        Self(proptest::test_runner::TestRng::from_seed(algorithm, seed))
+    }
+}
+
+#[cfg(test)]
+impl crate::private::Rng for ProptestRng {
+    fn random_bytes<const N: usize>(&self) -> [u8; N] {
+        use rand::RngCore;
+        let mut bytes = [0u8; N];
+        rand::thread_rng().fill_bytes(&mut bytes);
```

Comment:
Ah okay, I think there was a misunderstanding: My idea was to use `self.0.fill_bytes(&mut bytes)` here :v: (and that would require `self` to be `& mut`.
Let's just not do that for now, maybe we'll consider it for deterministic tests later.

Reply:
Wrote it wrong. Let me fix that.